app/components/modals/session-notify-modal.js

```diff
@@ -13,6 +13,12 @@ export default class SessionNotifyModal extends ModalBase {
   @tracked saving = false;
   @tracked subject = '';
   @tracked message = '';
+  @tracked cc = '';
+  
+  speakerEmails =  computed(function(){
```

Comment:
You didn't list the dependencies of the computed function

Reply:
I have done it. You can please see the changes made.